File: app/src/main/java/com/bhm/sdk/demo/tools/MyLoadingDialog.java
package com.bhm.sdk.demo.tools;


import com.microport.rxhttp.rxjava.RxBuilder;
import com.microport.rxhttp.utils.RxLoadingDialog;
import com.microport.rxhttp.utils.RxLoadingFragment;

/**
 * Created by bhm on 2018/5/14.
 */

public class MyLoadingDialog extends RxLoadingDialog {

    @Override
    public RxLoadingFragment initDialog(RxBuilder builder) {
        return new MyRxLoadingFragment(builder);
    }
}

File: app/src/main/java/com/bhm/sdk/demo/tools/MyLoadingDialog.kt
package com.bhm.sdk.demo.tools

import com.bhm.rxhttp.core.RxBuilder
import com.bhm.rxhttp.base.RxLoadingDialog
import com.bhm.rxhttp.base.RxLoadingFragment

/**
 * Created by bhm on 2022/9/15.
 */
class MyLoadingDialog : RxLoadingDialog() {
    override fun initDialog(builder: RxBuilder?): RxLoadingFragment {
        return MyRxLoadingFragment(builder!!)
    }
}